code/modules/client/preference_setup/traits/trait_defines.dm

```diff
@@ -0,0 +1,212 @@
+	qdel(M)
+	return new_desc
+
+
+// Physical traits are what they sound like, and involve the character's physical body, as opposed to their mental state.
+/datum/trait/modifier/physical
+	category = "Physical"
+
+
+/datum/trait/modifier/physical/flimsy
+	name = "Flimsy"
+	desc = "You're more fragile than most, and have less of an ability to endure harm."
+	modifier_type = /datum/modifier/trait/flimsy
+	muturally_exclusive = list(/datum/trait/modifier/physical/frail)
+
+
+/datum/trait/modifier/physical/frail
+	name = "Frail"
+	desc = "Your body is very fragile, and has even less of an ability to endure harm."
+	modifier_type = /datum/modifier/trait/frail
+	muturally_exclusive = list(/datum/trait/modifier/physical/flimsy)
+
+
+/datum/trait/modifier/physical/haemophilia
+	name = "Haemophilia"
+	desc = "Some say that when it rains, it pours.  Unfortunately, this is also true for yourself if you get cut."
+	modifier_type = /datum/modifier/trait/haemophilia
+
+/datum/trait/modifier/physical/haemophilia/test_for_invalidity(var/datum/category_item/player_setup_item/traits/setup)
+	if(setup.is_FBP())
+		return "Full Body Prosthetics cannot bleed."
+	// If a species lacking blood is added, it is suggested to add a check for them here.
+	return ..()
+
+/datum/trait/modifier/physical/weak
+	name = "Weak"
+	desc = "A lack of physical strength causes a diminshed capability in close quarters combat."
+	modifier_type = /datum/modifier/trait/weak
+
+
+/datum/trait/modifier/physical/inaccurate
+	name = "Inaccurate"
+	desc = "You're rather inexperienced with guns, you've never used one in your life, or you're just really rusty.  \
+	Regardless, you find it quite difficult to land shots where you wanted them to go."
+	modifier_type = /datum/modifier/trait/inaccurate
+
+
+// These two traits might be borderline, feel free to remove if they get abused.
+/datum/trait/modifier/physical/high_metabolism
+	name = "High Metabolism"
+	modifier_type = /datum/modifier/trait/high_metabolism
+
+/datum/trait/modifier/physical/high_metabolism/test_for_invalidity(var/datum/category_item/player_setup_item/traits/setup)
+	if(setup.is_FBP())
+		return "Full Body Prosthetics do not have a metabolism."
+	return ..()
+
+
+/datum/trait/modifier/physical/low_metabolism
+	name = "Low Metabolism"
+	modifier_type = /datum/modifier/trait/low_metabolism
+
+/datum/trait/modifier/physical/low_metabolism/test_for_invalidity(var/datum/category_item/player_setup_item/traits/setup)
+	if(setup.is_FBP())
+		return "Full Body Prosthetics do not have a metabolism."
+	return ..()
+
+
+/datum/trait/modifier/physical/cloned
+	name = "Cloned"
+	desc = "At some point in your life, you died and were cloned."
+	modifier_type = /datum/modifier/cloned
+
+/datum/trait/modifier/physical/cloned/test_for_invalidity(var/datum/category_item/player_setup_item/traits/setup)
+	if(setup.is_FBP())
+		return "Full Body Prosthetics cannot be cloned."
```

Comment:
You can put a cloned brain in an FBP

Reply:
If it eventually has a mechanical effect other than acting as a death counter, we'd rather not have it affect FBPs.